### DISCUSSION TITLE
From where I can download windows installer?

### DISCUSSION THREAD

**Discussion by salmanmalik-emb** [strongswan/strongswan#1674] [Q&A]
I am looking for windows installer. I am unable to find it anywhere in the documentation. The previous link I have used once is broken as well.

https://download.strongswan.org/windows/strongswan-5.9.3.zip

**Comment by tobiasbrunner** [comment]
>  I am looking for windows installer. 

There is no Windows installer for strongSwan (see the docs for [details on the Windows port](https://docs.strongswan.org/docs/5.9/os/windows.html)).

> The previous link I have used once is broken as well.

No idea where you got that link from because there never was a `windows` folder on our download host.

**Comment by salmanmalik-emb** [comment]
Hi @tobiasbrunner ,

Thanks for the response. I am following this https://docs.strongswan.org/docs/5.9/os/windows.html to build for windows. 

Within the following command, what should be the value for openssl /path/to/openssl/include and /path/to/openssl/lib, I guess this is openssl source folder. I have download openssl source from https://github.com/openssl/openssl but it doesnt have any lib folder.


`CFLAGS="-g -O2 -Wall -Wno-pointer-sign -Wno-format-security \
        -Wno-format -mno-ms-bitfields \
        -I/c/path/to/openssl/include" \
LDFLAGS="-L/c/path/to/openssl/lib" \
./configure --disable-defaults --enable-monolithic --enable-static \
            --enable-svc --enable-ikev2 --enable-ikev1  \
            --enable-nonce --enable-pem --enable-pkcs1 \
            --enable-x509 --enable-openssl --enable-socket-win \
            --enable-kernel-wfp --enable-kernel-iph --enable-pubkey \
            --enable-swanctl --with-swanctldir=swanctl \
            --with-strongswan-conf=strongswan.conf`
            
            

Therefore, while running above command I am getting   following error.
`checking for EVP_CIPHER_CTX_new in -leay32... no
no
checking for EVP_CIPHER_CTX_new in -lcrypto... no
configure: error: OpenSSL libcrypto not found`

Can you please guide me on this?

**Reply by tobiasbrunner** [reply]
> what should be the value for openssl /path/to/openssl/include and /path/to/openssl/lib, I guess this is openssl source folder.

No, the path to the installed OpenSSL library and header files (you can use a binary build, there are [different ones for Windows](https://wiki.openssl.org/index.php/Binaries) - I guess you could also build OpenSSL from sources, but that's not necessary).
